src/vorta/scheduler.py

```diff
@@ -30,6 +32,11 @@ def __init__(self, parent):
         self.qt_timer.setInterval(45 * 60 * 1000)
         self.qt_timer.start()
 
+    def cancel_all_jobs(self):
```

Comment:
Depending on where this is used, the caller could just use `self.jobs_manager.cancel_all_jobs()` directly, since it's not really related to the scheduler.

Reply:
I am not sure about this. This method simply cancels all jobs without knowing the scheduler in depth. Calling cancel_all_jobs directly, some method will call "this_instance.scheduler.jobs_manager.cancel_all_jobs()" which is long. If we do that, maybe the JobsManager can be initialized in the VortaApp and the previous call becomes "this_instance.jobs_manager.cancel_all_jobs()".